### DISCUSSION TITLE
www.ohmygpt.com三方提供API无法使用

### DISCUSSION THREAD

**Discussion by tbs575** [ChatGPTNextWeb/NextChat#4709] [Q&A]
采用Docker部署的Nextweb环境，采用 www.ohmygpt.com 三方网站提供的API，能正常使用 GPT 模型。
但使用claude-3相关模型提示错误
```
{
  "statusCode": 401,
  "message": "Incorrect API key provided. You can get your API key at https://www.ohmygpt.com/settings",
  "data": null
}
```
直接采用https://www.ohmygpt.com/这个网站提供会话，是能使用claude-3模型的。是需要在docker设置什么参数么？

**Comment by DINGBROK423** [comment]
我也遇到相同问题

<img width="911" height="661" alt="e3468874258eb9f84aa46531c7d45613" src="https://github.com/user-attachments/assets/2a8ed1a8-5d0d-4831-bbb5-9371cdc069cf" />
